sources/packages/workflow/BPMN/camunda-8/assessment-2021-2022.bpmn

```diff
@@ -1365,7 +1365,7 @@ or (calculatedDataPartnerFedralProvincialPDReceiptCost &gt; 0)</bpmn:conditionEx
     <bpmn:startEvent id="StartEvent_1" name="start">
       <bpmn:extensionElements>
         <zeebe:ioMapping>
-          <zeebe:output source="=institutionLocationProvince" target="institutionLocationProvince" />
+          <zeebe:output source="=if is defined(institutionLocationProvince) = false then null else institutionLocationProvince" target="institutionLocationProvince" />
```

Comment:
Minor but let's review this logic that is used in more than one place.
If this is to allow the IntelliSense when a variable is also possibly received, other points in the workflow declare the variables at the start event like below:
![image](https://user-images.githubusercontent.com/61259237/202537680-fbadbb30-3157-4e68-825c-32c29ac494be.png)

Reply:
removed due to #1554